src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -399,6 +400,96 @@ public Task GetImport()
             }
         }
 
+        [RavenAction("/databases/*/smuggler/import/csv", "POST", AuthorizationStatus.ValidUser)]
+        public async Task ImportFromCsv()
+        {
+            using (Database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+            {
+                if (HttpContext.Request.HasFormContentType == false)
+                {
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                    using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                    {
+                        context.Write(writer, new DynamicJsonValue
+                        {
+                            ["Type"] = "Error",
+                            ["Error"] = "Import from csv requires form content type"
+                        });
+                        return;
+                    }
+                }
+                var token = new OperationCancelToken(Database.DatabaseShutdown);
+                var result = new SmugglerResult();
+                var operationId = Database.Operations.GetNextOperationId();
+                await Database.Operations.AddOperation(Database, $"Import to: {Database.Name} from csv file", Raven.Server.Documents.Operations.Operations.OperationType.DatabaseImportFromCsv,
```

Comment:
I'm not databasename in task name is required. We already have this information in context.

Reply:
okay